packages/app/src/TeamPage.tsx

```diff
@@ -672,12 +693,17 @@ function CreateWebhookForm({
   });
 
   const onSubmit: SubmitHandler<WebhookForm> = async values => {
+    const { service, name, url, description, body } = values;
     try {
       await saveWebhook.mutateAsync({
-        service: values.service,
-        name: values.name,
-        url: values.url,
-        description: values.description || '',
+        service,
+        name,
+        url,
+        description: description || '',
+        body:
+          service === 'generic' && !body
```

Comment:
style: can use enum here https://github.com/hyperdxio/hyperdx/blob/4586cc1c1bf54b00dbfcea855a0c1e19176dc237/packages/common-utils/src/types.ts#L168

Reply:
sure, let me update it